erreser.go

```
@@ -0,0 +1,666 @@
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/subcommands"
+	"github.com/mmcdole/gofeed"
+	"gopkg.in/telebot.v3"
+	"gorm.io/gorm"
+)
+
+func getEnvWithDefault(key, defaultValue string) string {
```

golangci-lint (1.62.2)

func `getEnvWithDefault` is unused

(unused)